simplified approach/mac.md

```diff
@@ -120,27 +120,54 @@ sunodo run
 
 ### Sending inputs to the application
 
-Your applications can receive inputs by sending transactions with the input payload.
+With your application running, you can send inputs by sending transactions with the input payload.
 
-To send inputs, use the command:
 
-```
+You can send inputs to your application using the sunodo send command, Cast, a command-line tool for making Ethereum RPC calls, or a  custom-built web interface.
```

Comment:
To send inputs to your application, you have a few options available. The sunodo send command is one option that you can use. Another option is Cast, which is a command-line tool that enables you to make Ethereum RPC calls. Additionally, you can build a custom web interface to input data into your application.